src/graph/Node.hpp

```diff
@@ -131,6 +133,18 @@ struct Node : APIObject<Node>, std::enable_shared_from_this<Node>
 	 */
 	virtual void validateImpl() = 0;
 
+	/**
+	 * @return True, if node can be executed or queried for results.
```

Comment:
This may suggest, that we can query node without execution.

Reply:
Yes, this is utterly wrong.